tests/schema/tests.py

```diff
@@ -173,6 +173,19 @@ def assertIndexOrder(self, table, index, order):
         index_orders = constraints[index]['orders']
         self.assertTrue(all([(val == expected) for val, expected in zip(index_orders, order)]))
 
+    def assert_foreign_key_exists(self, model, column, expected_fk_details):
```

Comment:
It might be nice to add a docstring. In particular, I'm not sure if the value of `expected_fk_details` is self-explanatory. I see it's always something like `('table', 'id')`. I guess it's possible the second value isn't 'id' but I wonder if it's worth keeping it simple for now and just passing the table. Another option could be to change the signature to something like `expected_table, expected_column='id'` (names could probably be better).